packages/pages/src/upgrade/upgrade.ts

```diff
@@ -12,8 +25,20 @@ interface UpgradeArgs {
 
 const handler = async (args: UpgradeArgs) => {
   const scoped = { scope: args.scope || "" };
+  const source = path.resolve(scoped.scope);
   const projectStructure = await ProjectStructure.init(scoped);
-  await updatePages(projectStructure);
+  await updateSitesComponents(source);
```

Comment:
Can you move this one right before `updatePages` since it's the same thing? Actually, it'd be better to combine `updatePages` and `updatePagesComponents` to each do the two things: update the imports and update the package.json, since it's really one thing we're trying to update overall. The method can call 2 sub-methods.

Reply:
I confused `updatePages` because the name was not clear. At any rate, I mean to combine the updating of the sites-components package and updating all of the imports under one function.